drivers/SmartThings/matter-switch/src/init.lua

```diff
@@ -243,6 +428,8 @@ end
 
 local function device_init(driver, device)
   if device.network_type == device_lib.NETWORK_TYPE_MATTER then
+    device:set_component_to_endpoint_fn(component_to_endpoint)
```

Comment:
Can you remind me why these functions needed to be moved to the top of device_init? Just want to make sure I understand why this change was needed

Reply:
This was a leftover from when the deferred configuration for buttons wasn't in place, and I believe they can safely be restored to their original location.

Reply:
Restored to their original position in de641ad73df7965367fa355d0b54a3570ccec054